deps/rabbitmq_web_stomp/src/rabbit_web_stomp_handler.erl

```diff
@@ -291,7 +295,8 @@ handle_data1(Bytes, State = #state{proc_state  = ProcState,
                                      proc_state  = ProcState1,
                                      connection  = ConnPid });
                 {stop, _Reason, ProcState1} ->
-                    stop(State#state{ proc_state = ProcState1 })
+                    self() ! close_websocket,
+                    {ok, State#state{ proc_state = ProcState1 }}
```

Comment:
Add comment why not stopping here directly:
`%%  do not exit here immediately, because we need to wait for messages eventually enqueued by process_request`